ami/main/api/views.py

```diff
@@ -1050,23 +1050,41 @@ class OccurrenceTaxaListFilter(filters.BaseFilterBackend):
     """
 
     query_param = "taxa_list_id"
+    query_param_exclusive = f"not_{query_param}"
 
     def filter_queryset(self, request, queryset, view):
```

Comment:
For the BE implementation, I had a peak at the class `TagInverseFilter` and then just copied some code. Feel free to help me clean it up if needed, I might introduced some repetition! :)